hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -1101,6 +1105,10 @@ struct shlq { Vreg64 s, d; VregSF sf; Vflags fl; }; // uses rcx
 /*
  * arm intrinsics.
  */
+struct csoneb { ConditionCode cc; VregSF sf; Vreg8 t, d; };
```

Comment:
In vasm-fold-imms.cpp they're in a 'namespace arm'.  The other PhysRegs are either in
target specific or register specific files.   The only target specific code in vasm-simplify are the tests against arch().  There are Vinstrs labeled as intrinsics, so I thought it safe to
create one of those.  You have to pick something which avoids using the reg in the initial
move otherwise it won't be eliminated as dead code.  The pseudo instruction seemed like the right place to emit the xzr.